routes/global_challenge.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from flask import Blueprint, jsonify
+from flask_api import status
+from flask_jwt_extended import get_jwt_identity
+from models.user import User
+from util.database import db, get_engine
+from util.error import UNKNOWN_USER, exception_message
+from util.login import login_required
+
+
+logger = logging.getLogger(__name__)
+IndividualGlobalChallenge = Blueprint('IndividualGlobalChallenge',
+                                      __name__)
+CommunityGlobalChallenge = Blueprint('CommunityGlobalChallenge', __name__)
+
+
+def get_individual_saving(meter_id):
+    """ Retrieve the last individual saving prognosis for the given meter id
+    from the SQLite database.
+    :param str meter_id: the user's meter id
+    :returns: the last saving together with its timestamp or None if there are
+    no values
+    :rtype: dict or type(None) if there are no values
+    """
+
+    # pylint: disable=line-too-long
+    query = "SELECT timestamp, saving FROM user_saving WHERE meter_id = '%s' ORDER BY timestamp DESC" % meter_id
```

Comment:
``
```suggestion
    query = ("SELECT timestamp, saving FROM user_saving"
             "WHERE meter_id = '%s'"
             "ORDER BY timestamp DESC") % meter_id
```

Then you can get rid of the ` # pylint: disable=line-too-long`

Reply:
Did you try to use this with SQLAlchemy? It is not working, you get an OperationalError: 
```ERROR:routes.global_challenge:An exception of type OperationalError occurred. Arguments:
('(sqlite3.OperationalError) near "=": syntax error',)```
This is why I used this workaround in the first place.